common/lib/protocol-definitions/src/tokens.ts

```diff
@@ -66,23 +66,52 @@ export interface ITokenClaims {
     jti?: string;
 }
 
+/**
+ * TODO
+ */
 export interface ISummaryTokenClaims {
+    /**
+     * TODO
+     */
     sub: string;
+
+    /**
+     * TODO
+     */
     act: IActorClient;
+
+    /**
+     * TODO
+     */
     claims: ITokenClaims;
 }
 
+/**
+ * TODO
+ */
 export interface IActorClient {
+    /**
+     * TODO
+     */
     sub: string;
 }
 
 /**
  * Abstracts the discovery of claims contained within a token.
  */
 export interface ITokenService {
```

Comment:
Is this even used anywhere?

Reply:
Yeah, it's used in routerlicious

Reply:
Per the other discussion, this has been marked as deprecated.